.pre-commit-config.yaml

```
@@ -0,0 +1,56 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
```

Exclude confusables from all scans so as to make fetching from upstream easier if @ThioJoe ever wants to update that
```suggestion
exclude: Scripts/confusablesCustom/

repos:
```

---

LGTM✅